docker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.6
```

Comment:
@bclermont Just wanted to mention it as a possibilty for future work on this.
As of today 17.06 is in rc5 and should arrive soon(ish?) as next stable release.